package.json

```diff
@@ -185,6 +185,7 @@
     "@types/intl-tel-input": "17.0.4",
     "@types/react-datepicker": "4.4.1",
     "@xstyled/emotion": "2.5.0",
+    "babel-plugin-named-exports-order": "^0.0.2",
```

Comment:
devDeps maybe ?

Reply:
We can also move the above `@types` packages to devdeps

Reply:
Nope, @types/intl-tel-input and @types/react-datepicker are exported in the definition bundle :)

Reply:
Done